src/pages/Storefront/Cart.js

```diff
@@ -0,0 +1,119 @@
+import { CartItem } from 'components/storefront';
+import { Box, HStack, VStack, Heading, Divider, Flex, FormControl, Text, Button, Input } from '@chakra-ui/react';
+// import { useSelector, shallowEqual } from 'react-redux';
+// import { selectors as teamSelectors } from '../../data/reducers/team';
+
+// Cart page that shows the user the items in their cart, allows them to change quantity, and proceed to checkout
+
+const Cart = () => {
+  const { products, checkoutState } = useShopify();
+  // const team = useSelector(teamSelectors.selectTeam, shallowEqual);
+
+  /* previous code that might be userful
+  useEffect(() => {
+    const userID = sessionStorage.getItem('userID');
+    updateCartAttributes(checkoutState.id, [
+      { key: 'userID', value: userID ? userID.toString() : '1' }, // Temporary, so that page doesn't crash
+      // Remove once user info is added to redux
+      { key: 'teamID', value: team.id.toString() },
+      { key: 'teamName', value: team.name },
+    ]);
+  }, [checkoutState.id, team.id, team.name]); // eslint-disable-line react-hooks/exhaustive-deps
+  */
+
+  // Temporary checkout code from last term, will need to udpate
+  const openCheckout = () => {
+    if (checkoutState.webUrl) {
+      console.log('webUrl exists');
+    }
+    // window.open(checkoutState.webUrl) // opens checkout in a new window
+    window.location.replace(checkoutState.webUrl); // opens checkout in same window
+  };
+
+  const applyCoupon = () => {
+    alert('To be implemented');
+  };
+
+  return (
+    <>
+      <Box bg="black" color="black" px="105px">
+        <Heading alignSelf="flex-start" color="white" py={6}>
+          MY SHOPPING BAG
+        </Heading>
+      </Box>
+      <HStack w="100%" h="100%" justifyContent="space-between" alignItems="flex-start" px="105px">
+        <VStack w="100%" alignItems="flex-start" py="64px" pr="45px">
+          <Flex w="100%" justifyContent="space-between">
+            <Heading as="h4" size="subtitle" color="brand.black">
+              CONTINUE SHOPPING
+            </Heading>
+            <Heading as="h4" color="brand.gray" size="subtitle" textTransform="uppercase">
+              {/* TO DO: update this to be the length of the line items in cart */}
+              {`${products.length} ITEMS`}
+            </Heading>
+          </Flex>
+          <Divider borderColor="brand.gray" />
+          {/* TO DO: The products here should be the line-items in the Shopify Checkout, also need to add unique key */}
+          {products &&
+            products.map(product => (
+              <Box w="100%" pt="28px">
+                <CartItem key={product.id} product={product} />
+                <Divider borderColor="brand.gray" pb="36px" />
+              </Box>
+            ))}
+          <HStack py="30px">
+            <FormControl w="40%">
+              <Input
+                type="coupon"
+                name="coupon"
+                placeholder="123ABCD"
```

Comment:
I think the 123ABCD was an example of an input but the placeholder should be `COUPON CODE` - not entirely sure so probably confirm with designers.
```suggestion
                placeholder="COUPON CODE"
```

Reply:
this would probably make sense